.github/workflows/README.md

```diff
@@ -0,0 +1,248 @@
+# Workflows
+
+## `benchmarks.yml`
+
+<p>
+This is a GitHub Actions workflow that benchmarks the performance of MiGraphX, a machine learning framework, using the Open Neural Network Exchange Runtime (ONNX Runtime) framework.
+</p>
+
+- ## Trigger
+Note: Right now its not working because [benchmark.yaml](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/.github/workflows/benchmark.yaml) workflow needed to be updated.
+> The workflow will be triggered on workflow dispatch event from caller workflow [benchmark.yaml](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/.github/workflows/benchmark.yaml)
+
+- ## Input Parameters
+The workflow uses the following input parameters:
+
+> - `rocm_version`: The ROCm release version, which is a required parameter.
```

Comment:
The ">" character creates a visual grey bar to the left of the bullet.  Is there any reason to do that? The other file in this PR does not have it and it looks better.  The grey bar looks like a change history tag

Reply:
As we discussed at the meeting, now it's changed how we agreed to be.